index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+
```

Comment:
I think its better to remove the extra free space in your code, as it creates confusion rather its better implementing it in the right time and place.👍